.gitignore

```diff
@@ -192,3 +192,4 @@ Temporary Items
 
 #Visual studio stuff
 *.vscode/*
+html/changelogs/AutoChangeLog-pr-30416.yml
```

Comment:
Sir you have made a mistake here

Reply:
wat

Reply:
Why the hell do these files keep popping up, random ass files that I have done nothing to